src/pages/dar_collection_review/DarCollectionReview.js

```diff
@@ -82,6 +83,7 @@ export default function DarCollectionReview(props) {
       const collection = await Collections.getCollectionById(collectionId);
       const { dars, datasets } = collection;
       const darInfo = find((d) => !isEmpty(d.data))(collection.dars).data;
+      const referenceIdForDocuments = find((d) => !isEmpty(d.referenceId))(collection.dars).referenceId;
```

Comment:
In speaking with @JVThomas, any one of the collection of DARs should point to the proper referenceId that is needed to obtain the documents from the v2 endpoint.

Reply:
That is correct. Alternatively, we can add parallel APIs at the collection level that do the same thing. I'm fine with this approach.